src/main/java/org/folio/inventory/dataimport/handlers/actions/CreateInstanceEventHandler.java

```diff
@@ -105,6 +105,16 @@ public boolean isEligible(DataImportEventPayload dataImportEventPayload) {
     return false;
   }
 
+  @Override
+  public boolean isPostProcessingNeeded() {
+    return true;
+  }
+
+  @Override
+  public String getPostProcessingInitializationEventType() {
+    return "DI_INVENTORY_INSTANCE_CREATED_READY_FOR_POST_PROCESSING";
```

Comment:
I think it will be better to use enum here, not simple String.

Reply:
sure, I need to merge processing-core PR to be able to use event types enum.